src/pages/execution/workflowTabs/WorkflowAlerts.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext, useEffect, useMemo } from "react";
+import Stack from "@mui/material/Stack";
+import { rules, Severity, AlertItem } from ".//ExpertSystemRules";
+import { ExecutionAndTasks } from "../../../types/execution";
+
+export default function WorkflowAlerts({ executionAndTasks, setSeverity }: { executionAndTasks: ExecutionAndTasks, setSeverity: Function }) {
+  const alerts = useMemo(() => {
+    const allAlerts: AlertItem[] = [];
+
+    rules.forEach((rule) => {
```

Comment:
Lets wrap this block with a ```
try {
 <check rules...>
 } catch (e) {
   console.log("error evaluating rules", e);
}
```
In order to isolate any future rule errors from breaking the UI render.